noir/tooling/noir_js_backend_barretenberg/src/index.ts

```diff
@@ -143,31 +111,13 @@ export class BarretenbergBackend implements Backend {
     };
   }
 
-  async verifyFinalProof(proofData: ProofData): Promise<boolean> {
+  /** @description Verifies a proof */
+  async verifyProof(proofData: ProofData): Promise<boolean> {
     const proof = reconstructProofWithPublicInputs(proofData);
-    const makeEasyToVerifyInCircuit = false;
-    const verified = await this.verifyProof(proof, makeEasyToVerifyInCircuit);
-    return verified;
-  }
-
-  /**
-   *
-   * @example
-   * ```typescript
-   * const isValidIntermediate = await backend.verifyIntermediateProof(proof);
-   * ```
-   */
-  async verifyIntermediateProof(proofData: ProofData): Promise<boolean> {
-    const proof = reconstructProofWithPublicInputs(proofData);
-    const makeEasyToVerifyInCircuit = true;
-    return this.verifyProof(proof, makeEasyToVerifyInCircuit);
-  }
-
-  /** @ignore */
-  async verifyProof(proof: Uint8Array, makeEasyToVerifyInCircuit: boolean): Promise<boolean> {
     await this.instantiate();
     await this.api.acirInitVerificationKey(this.acirComposer);
-    return await this.api.acirVerifyProof(this.acirComposer, proof, makeEasyToVerifyInCircuit);
+    // TODO: Change once `@aztec/bb.js` version is updated to use methods without isRecursive flag
```

Comment:
I updated these methods but perhaps we should instead revert the changes here for NoirJS until we have an updated release of `bb.js` with the breaking changes.